pkg/batchd/cache/cache.go

```diff
@@ -88,6 +89,10 @@ func newSchedulerCache(config *rest.Config) *SchedulerCache {
 			FilterFunc: func(obj interface{}) bool {
 				switch t := obj.(type) {
 				case *v1.Pod:
+					pod := obj.(*v1.Pod)
+					if strings.Compare(pod.Spec.SchedulerName, schedulerName) == 0 && pod.Status.Phase == v1.PodPending {
```

Comment:
it also adds pending pods from other scheduler(`nonTerminatedPod()` will return true)

Reply:
ah I see, will fix it soon, thanks